### ISSUE TITLE
Interface not showing&not clearing existing binds

### ISSUE THREAD

**Issue by zulubravo0** [Holdi601/JoystickProfiler#8]
My experience after few days:
-Start program few times, it works nice
  after few days, I start my pc, click on icon, icon of the active program appears on the taskbar but there is no window of the program interface on my desktop, I try to alt-tab it to the desktop and it is unable to show itself. In the alt-tab window it looks like simple window but filled with white color(sometimes black).
![joypro1](https://user-images.githubusercontent.com/82122085/113936576-3f9efe00-97f8-11eb-9716-5b95debcf0b3.png)

-Second issue is that program doesn't clear existing binds when you press the button and sometimes it even adds some of the bindings you never bind anywhere.
example, I load relation file template Johnny and bind some buttons and axes, I press "clean existing and...", action complete, I enter DCS and I cannot control anything because there is a new binding which I haven added, warthog h.o.t.a.s. mini joystick is bind to pitch and yaw or maybe rudders.
It seems, maybe, If I bind enough keys and axes that problem disappears.
that is my experience, I don't know what is the cause. 
Cheers!
p.s. 3 windows on the image just means I tried to open it 3 times and it opened 3 times but window is not available

**Comment by Holdi601** [comment]
issue 1:
Delete the folder Documents\JoyPro there are some window state files saved in like profiles. It sounds like it got corrupted and it tries to reload it. delete it see if that makes it work.

issue 2: 
if you dont have checked nuke other devices that you dont use, DCS will still apply default bindings to those USB Joystick devices which have no binds in Joy pro. So if you made just some binds to your joystick but none to your throttle, without that one checked it will only write the stick you bound stuff to. With the nuke checkbox, it writes every connected device clean regardless if you have bound anything to it or not.

issue 3: 
see 1 delete that folder if it got corrupted or it corrupted saved it, it may cause it. Delete the joypro folder in documents and restart the program.

**Comment by zulubravo0** [comment]
Thank you very much, I will try this tomorrow!

**Comment by zulubravo0** [comment]
Ok, I have tried this, It works now.
I have few modules (mi8,FC3,KA50,P51) , for some reason there are key binds in mi8 that I haven't put there ever in any way. I had previously mapped mi8 and it had a lot of keys mapped before joypro, I used nuke option.
1. example, joypro bind button 7 and button 8 to free turbine RPM, but I did not do that I put button 7 and button 8 as modifiers.
2. example: I haven't bind left and right external light (moving up/down/left/right) to the throttle hat switches, it appeared by itself.
3. example: I bind button 1 on joystick to  trimmer button (on mi8), when I enter controls menu in DCS somethin really weird happens, When I press button 1 it shows Radio trigger, then I press again but it jumps to Trimmer button, and if I keep pressing the button it alternates between 2 controls, I have never seen that in DCS because it is impossible to bind same button to different things.
before I used joypro I had left and right external light (moving up/down/left/right) mapped to joystick with modifiers
now with joypro they appeared on my throttle. I haven't noticed problems with other modules other than that I should be careful when adding relations, they have different names on different modules, on some modules trim works but on others not then I edit relation and it works. I used my own relations.
![dcs jp](https://user-images.githubusercontent.com/82122085/113955113-6de20500-981b-11eb-9e3c-d629bf3ba078.png)


EDIT1:
1. I have found that there is a problem with ka-50 also. It is a same problem Mi-8 has, there are a lot of button binds that I haven't put there. Even if I delete all bindings inside the DCS, save, exit, apply joypro profile(Clear Existing Binds...and Nuke... option), then start DCS, check controls, and there it is, a lot of button mapped that I haven't mapped.
